test/parallel/test-fs-watchfile.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const common = require('../common');
+const fixtures = path.join(__dirname, '..', 'fixtures');
+
+function noop() {}
+
+var callbackFired = 0;
+
+// Basic usage tests.
+assert.throws(function() {
+    fs.watchFile('./some-file');
+}, /watchFile requires a listener function/);
+
+assert.throws(function() {
+    fs.watchFile('./another-file', {}, 'bad listener');
+}, /watchFile requires a listener function/);
+
+assert.throws(function() {
+    fs.watchFile(new Object(), function() {})
+}, /path must be a string/i);
+
+// Test ENOENT. Should fire once.
+var enoentFile = path.join(fixtures, 'empty', 'non-existent-file');
+fs.watchFile(enoentFile, function(curr, prev) {
+    callbackFired++;
```

Comment:
Yes, it's the inode number.  `curr.ino > 0` is just an example, though; on error, all the fields will be zero.  It would be good to document that behavior in the `fs.watchFile()` documentation.